README.md

```diff
@@ -33,6 +33,26 @@ All JavaScript code should be written in `onDeviceReady` function in `/js/index.
     }
 ```
 
+## Positioning the camera's behind or before webview
```

Comment:
@mark-veenstra How do you feel about adding this information in the docs folder instead of the main repo readme?

Reply:
That's probably better. So I moved it to the docs folder.